lib/mongo/url_parser.ex

```diff
@@ -165,6 +165,22 @@ defmodule Mongo.UrlParser do
     end
   end
 
+  defp build_params(orig_options, txt_record) do
+    [orig_options, txt_record, "ssl=true"]
+    |> Enum.filter(&(&1 != nil))
+    |> Enum.join("&")
+  end
```

Comment:
It looks good to me. Since there are only two use cases, you could rewrite it like this:

```
defp build_params(orig_options, nil) do
   "#{orig_options}&ssl=true"
 end
  
defp build_params(orig_options, txt_record) do
"#{orig_options}&#{txt_record}&ssl=true"
end
```